tox.cover.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+import os
+import sys
+
+DEFAULT_PACKAGES = [
+    'certbot', 'acme', 'certbot_apache', 'certbot_dns_cloudflare', 'certbot_dns_cloudxns',
+    'certbot_dns_digitalocean', 'certbot_dns_dnsimple', 'certbot_dns_dnsmadeeasy',
+    'certbot_dns_gehirn', 'certbot_dns_google', 'certbot_dns_linode', 'certbot_dns_luadns',
+    'certbot_dns_nsone', 'certbot_dns_ovh', 'certbot_dns_rfc2136', 'certbot_dns_route53',
+    'certbot_dns_sakuracloud', 'certbot_nginx', 'certbot_postfix', 'letshelp_certbot']
+
+COVER_THRESHOLDS = {
+    'certbot': 98,
+    'acme': 100,
+    'certbot_apache': 100,
+    'certbot_dns_cloudflare': 98,
+    'certbot_dns_cloudxns': 99,
+    'certbot_dns_digitalocean': 98,
+    'certbot_dns_dnsimple': 98,
+    'certbot_dns_dnsmadeeasy': 99,
+    'certbot_dns_gehirn': 97,
+    'certbot_dns_google': 99,
+    'certbot_dns_linode': 98,
+    'certbot_dns_luadns': 98,
+    'certbot_dns_nsone': 99,
+    'certbot_dns_ovh': 97,
+    'certbot_dns_rfc2136': 99,
+    'certbot_dns_route53': 92,
+    'certbot_dns_sakuracloud': 97,
+    'certbot_nginx': 97,
+    'certbot_postfix': 100,
+    'letshelp_certbot': 100
+}
+
+def cover(package):
+    threshold = COVER_THRESHOLDS[package]
+
+    if not threshold:
+        raise ValueError('Unrecognized package: {0}'.format(package))
```

Comment:
I don't think we'll ever hit this line. If `package` isn't in `COVER_THRESHOLDS`, won't we blow up with a `KeyError`? Maybe we should use `COVER_THRESHOLDS.get` above.

Reply:
Indeed. Corrected.